enhancements/microshift/rhoai-model-serving-on-microshift.md

```diff
@@ -0,0 +1,339 @@
+- Don't include SRs in any of the RPM. Instead include them in the documentation
+  for users to copy and include in their manifests.
+  - This might be prone to getting outdated very easily as documentation is not
+    part of the MicroShift rebase procedure.
+
+
+### No MicroShift support for GPU Operator, Node Feature Discovery, other hardware-enabling Operators, etc...
+
+[RHOAI's how to on using Raw Deployment](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.16/html/serving_models/serving-large-models_serving-large-models#deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode_serving-large-models) lists some requirements such as:
+> - If you want to use graphics processing units (GPUs) with your model server, you have enabled GPU support in OpenShift AI.
+> - To use the vLLM runtime, you have enabled GPU support in OpenShift AI and have installed and configured the Node Feature Discovery operator on your cluster
+
+~~Neither GPU Operator, NFD, or Intel Gaudi AI Accelerator Operator are supported on MicroShift.~~
+[Accelerating workloads with NVIDIA GPUs with Red Hat Device Edge](https://docs.nvidia.com/datacenter/cloud-native/edge/latest/nvidia-gpu-with-device-edge.html)
+is a procedure on setting up NVIDIA GPU for MicroShift. Users will be directed to
+that website's relevant sections and also to [RHOAI's guide on adding NVIDIA Triton ServingRuntime](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.16/html/serving_models/serving-large-models_serving-large-models#adding-a-tested-and-verified-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models).
+
+~~Should users be instructed to use upstream releases of these components and configure them on their own?~~
+
+If there's no existing guide on setting specific accelerator for usage with RHDE,
+we shall work with partners to achieve that. We want to avoid directing users
+to generic upstream information without any support.
+
+### Do we need ODH Model Controller?
+
+RHOAI Operator deploys both kserve and ODH Model Controller.
+In the catalog image for Model Controller is described as:
+> The controller removes the need for users to perform manual steps when deploying their models
+
+
+## Versioning
+
+While it might be best to have RPM with RHOAI version, because RHOAI does not
+follow OpenShift's release schedule and the RPM will live in the MicroShift
+repository, it will be versioned together with MicroShift.
+
+It means that certain version of RHOAI's kserve will be bound to MicroShift's
+minor version.
+
+
+## Test Plan
+
+At the time of writing this enhancement, RHOAI is not yet supported on ARM
+architecture, so testing on NVIDIA Jetson Orin devices will be postponed.
+
+First, a smoke test in MicroShift's test harness:
+- Stand up MicroShift device/cluster
+- Install kserve
+- Create an InferenceService using some basic model that can be easily handled by CPU
```

Comment:
Ok if this test case is CPU only I would cover only OpenVINO

Reply:
Is there something lighter just for smoke test that everything deploys and runs?